test/testother.cpp

```diff
@@ -2363,7 +2380,7 @@ class TestOther : public TestFixture {
               "    static int& i = x[0];\n"
               "    return i;\n"
               "}");
-        ASSERT_EQUALS("", errout.str());
```

Comment:
Here we show a warning although `i` also needs to be changed. Acceptable?

Reply:
No sure.

But it looks like code which applies to the suggested https://trac.cppcheck.net/ticket/11565.

Reply:
We actually detect the a `danglingReference` in the test case.